examples/CampaignManagement/AddCompleteCampaignsUsingMutateJob.php

```diff
@@ -0,0 +1,483 @@
+            printf(
+                "Request with ID '%s' has failed.%sGoogle Ads failure details:%s",
+                $googleAdsException->getRequestId(),
+                PHP_EOL,
+                PHP_EOL
+            );
+            foreach ($googleAdsException->getGoogleAdsFailure()->getErrors() as $error) {
+                /** @var GoogleAdsError $error */
+                printf(
+                    "\t%s: %s%s",
+                    $error->getErrorCode()->getErrorCode(),
+                    $error->getMessage(),
+                    PHP_EOL
+                );
+            }
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the client customer ID without hyphens
+     */
+    public static function runExample(GoogleAdsClient $googleAdsClient, $customerId)
+    {
+        $mutateJobServiceClient = $googleAdsClient->getMutateJobServiceClient();
+
+        // Creates a new mutate job.
+        $mutateJobResourceName =
+            $mutateJobServiceClient->createMutateJob($customerId)->getResourceName();
+        printf(
+            "Created a mutate job with resource name: '%s'.%s",
+            $mutateJobResourceName,
+            PHP_EOL
+        );
+
+        // Adds all mutate job operations to the mutate job. As this is the first time for this
+        // mutate job, pass null as a sequence token.
+        $response = $mutateJobServiceClient->addMutateJobOperations(
+            $mutateJobResourceName,
+            null,
+            self::buildAllOperations($customerId)
+        );
+        printf(
+            "%d mutate operations have been added so far.%s",
+            $response->getTotalOperations(),
+            PHP_EOL
+        );
+        // You can use this next sequence token for calling addMutateJobOperations() next time.
+        printf(
+            "Next sequence token for adding next operations is '%s'.%s",
+            $response->getNextSequenceToken(),
+            PHP_EOL
+        );
+
+        // Runs the mutate job for executing the added operations.
+        $operationResponse = $mutateJobServiceClient->runMutateJob($mutateJobResourceName);
+        printf(
+            "Mutate job with resource name '%s' has been executed.%s",
+            $mutateJobResourceName,
+            PHP_EOL
+        );
+
+        // Polls the server until the mutate job execution finishes by setting the initial poll
+        // delay time and the total time to wait before time-out.
+        $operationResponse->pollUntilComplete([
+            'initialPollDelayMillis' => self::POLL_FREQUENCY_SECONDS * 1000,
+            'totalPollTimeoutMillis' => self::MAX_TOTAL_POLL_INTERVAL_SECONDS * 1000
```

Comment:
This is fine for now, but talk to Ben / Bob about whether we can autogenerate reasonable defaults for these parameters.